.editorconfig

```
@@ -0,0 +1,13 @@
+root=true
+
+[*]
+charset=utf-8
+insert_final_newline=true
+
+[LICENSE.md]
+indent_size=unset
+indent_style=space
+
+[*.{json,yml}]
```

Why not insert here `.js` extension files with 2 spaces of indentation and set that indentation in `build-testpage.js` and `version.js` scripts accordingly?

```suggestion
[*.{js,json,yml}]
```

---

I agree, but would prefer to have that refactored in a separate PR. I think the JavaScript can use some more refactoring to improve consistency. Perhaps we should look into using [Prettier](https://prettier.io/) in this project, thoughts?

---

Yes, Prettier or Eslint, I'm fine with both, but Prettier is easier to set up :+1: